### ISSUE TITLE
Wrong IMDb rating

### ISSUE THREAD

**Issue by trulex** [mrwrob/netfweb#15]
For movie [Solo](https://www.imdb.com/title/tt6699860/?ref_=nv_sr_5), rating and link will point to [Solo: A Star Wars Story](https://www.imdb.com/title/tt3778644/?ref_=fn_al_tt_1).
![image](https://user-images.githubusercontent.com/3431769/54481577-c14d1900-4836-11e9-9851-2cfafd19e2a4.png)

**Comment by metaxis** [comment]
similar problem:  I noted the rating for the Joss Whedon "Serenity" (2005) film was returning the much, much lower rating for the unrelated 2019 film of the same name.

**Comment by mrwrob** [comment]
Yeah... this is not the problem with the extension itself, but rather with the searching and matching. However, you can support us in fixing the mappings. Take a look at: https://www.youtube.com/watch?v=73eLVFmL0lo

**Comment by B3nji957** [comment]
> Yeah... this is not the problem with the extension itself, but rather with the searching and matching. However, you can support us in fixing the mappings. Take a look at: https://www.youtube.com/watch?v=73eLVFmL0lo

@mrwrob there is no longer the "x" button to fix the mappings.

**Comment by Javier-DarthPalpatine** [comment]
Hi @mrwrob
Please, can we have again the red "X" button or *any other way* to fix IMDB mappings?  (like showed in the video) 

Thanks!

**Comment by mrwrob** [comment]
@B3nji957 @Javier-DarthPalpatine, many thanks for bringing it to my attention! Red X" is not available due to the new Netflix layout. I will try to figure it out over the weekend.

**Comment by tekert** [comment]
Another problem with same name series (different year) Like Cowboy Beboop, showing 1999 scores on the 2021 live action adaptation.
Any progress?
